src/modules/assessments/components/alerts/AssessmentAlert.tsx

```diff
@@ -52,13 +55,18 @@ const AssessmentAlert: React.FC<Props> = ({
     color: 'text.primary',
     '.MuiAlert-icon': { alignItems: 'center', color: 'text.secondary' },
   };
+
+  // If an Icon is provided for the Alert, but there's also an Icon on the
+  // Button, then on mobile screens it looks better if we omit the Alert icon
+  const omitIcon = isTiny && action?.props.startIcon;
```

Comment:
Here's what it looks like without this line (with the double icon):
<img width="321" alt="AssessmentAlert-doubleicon" src="https://github.com/greenriver/hmis-frontend/assets/16002775/a0195d86-83d3-4840-aa06-5c640a8dc0bf">